src/server/components/dashboard/relatedLinks/deleteRelatedLink.ts

```diff
@@ -0,0 +1,39 @@
+import { prisma } from "server/models/db/prisma-client";
+import { logger } from "server/services/logger";
+import type { List } from "shared/types";
+
+export async function deleteRelatedLink(listId: List["id"], index: number | "new" = "new") {
```

Comment:
What does `"new" = "new"` do and why is that different from `"new"`?

Reply:
good spot - this shouldn't actually be there. 

`index: number | "new" = "new"`, parameter is called "index", and it can be a number, or a string that is "new". default to "new".